includes/db_connection.php

```diff
@@ -1,10 +1,14 @@
 <?php
-	// mysqli connection
-	//require_once('includes/settings.php');
```

Comment:
Are you talking about the commented code in line 3? Because I do not see any dead code. @MehaKaushik